packages/javascript/bh-shared-ui/src/components/Navigation/SubNav.test.tsx

```diff
@@ -26,11 +25,7 @@ describe('SubNav', () => {
             path: '/administration/test-nav-item-path',
         };
 
-        render(
-            <BrowserRouter>
```

Comment:
A Router is now in the set of default providers when using the test-utils render method